decidim-proposals/app/assets/javascripts/decidim/proposals/utils.js

```diff
@@ -0,0 +1,8 @@
+$("#vote_button").mouseover(function () {
```

Comment:
Please use the `.es6` extension on this file in order to transpile when necessary and also to avoid styling issues.